python.d/web_log.chart.py

```diff
@@ -808,15 +809,15 @@ def check(self):
             return False
         self.storage['unique_all_time'] = list()
         self.storage['regex'] = re.compile(r'[0-9.]+\s+(?P<duration>[0-9]+)'
-                                           r' (?P<client_address>[\da-f.:]+)'
+                                           r' (?P<client_address>[\dA-Za-z.:_-]+)'
                                            r' (?P<squid_code>[A-Z_]+)/'
                                            r'(?P<http_code>[0-9]+)'
                                            r' (?P<bytes>[0-9]+)'
                                            r' (?P<method>[A-Z_]+)'
                                            r' (?P<url>[^ ]+)'
                                            r' (?P<user>[^ ]+)'
-                                           r' (?P<hier_code>[A-Z_]+)/[\da-f.:-]+'
-                                           r' (?P<mime_type>[A-Za-z0-9-]+)')
+                                           r' (?P<hier_code>[A-Z_]+)/[\da-z.:-]+'
+                                           r' (?P<mime_type>[A-Za-z-]*)')
```

Comment:
`*` means that `mime_type` can be "" (empty `str`). I suggest to replace it with `+`.

Reply:
Well, i see 
`Also, some weird replies have content types ":" or even empty ones.`

https://wiki.squid-cache.org/Features/LogFormat

So `*` is ok.

Looks like we need to add empty line check to
https://github.com/firehol/netdata/blob/master/python.d/web_log.chart.py#L885-L886

Reply:
> Looks like we need to add empty line check to

`func_dim_id` will be "unknown" with empty mime-type returned by regexp